srcpkgs/libltc/template

```diff
@@ -0,0 +1,23 @@
+# Template file for 'libltc'
+pkgname=libltc
+version=1.3.1
+revision=1
+build_style=gnu-configure
+short_desc="Linear/Logitudinal Time Code (LTC) Library"
+maintainer="bluntphenomena <gregwyd@gmail.com>"
+license="GPL-3.0"
```

Comment:
needs spdx identifier

Reply:
ok how do i change this correctly without making too much commits or pull requests?
i cant figure out how to overwrite this commit after i made already the pull request